packageurl.go

```diff
@@ -120,6 +121,15 @@ func (q Qualifier) String() string {
 // in the package URL.
 type Qualifiers []Qualifier
 
+// urlQuery returns a raw URL query with all the qualifiers as keys + values.
+func (q Qualifiers) urlQuery() (rawQuery string) {
```

Comment:
You define `rawQuery` as return value, but I do not see it in the function. Is this a mistake?

Reply:
Simply for documentational reasons - the code doesn‘t need it. It‘s nice to just read the function header and know what it does.

that said - I‘m not particularly attached to it, I can remove it if you prefer :)